rotkehlchen/tests/unit/decoders/accounting_settings/test_default_settings.py

```diff
@@ -0,0 +1,271 @@
+from unittest.mock import patch
+
+import pytest
+
+from rotkehlchen.accounting.cost_basis.base import (
+    AssetAcquisitionEvent,
+    CostBasisInfo,
+    MatchedAcquisition,
+)
+from rotkehlchen.accounting.mixins.event import AccountingEventType
+from rotkehlchen.accounting.pnl import PNL
+from rotkehlchen.accounting.pot import AccountingPot
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.base import HistoryBaseEntry
+from rotkehlchen.accounting.structures.processed_event import ProcessedAccountingEvent
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants import ZERO
+from rotkehlchen.constants.assets import A_ETH
+from rotkehlchen.constants.misc import ONE
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.history import prices
+from rotkehlchen.types import Location, Timestamp, TimestampMS, deserialize_evm_tx_hash
+
+
+@pytest.fixture(name='pot')
+def fixture_pot(accountant):
+    pot: 'AccountingPot' = accountant.pots[0]
+    with pot.database.conn.read_ctx() as cursor:
+        settings = pot.database.get_settings(cursor)
+    pot.reset(
+        settings=settings,
+        start_ts=Timestamp(0),
+        end_ts=Timestamp(100),
+        report_id=1,
+    )
+    return pot
+
+
+def gain_one_ether(transactions, evmhash):
```

Comment:
Why do you need to pass the evmhash? is it used at some point?. The event is constant so it doesn't seem to be relevant